CMakeLists.txt

```diff
@@ -166,77 +174,74 @@ if(UMF_BUILD_EXAMPLES)
         message(STATUS "Found clang-format: ${CLANG_FORMAT} "
-                "(version: ${CLANG_FORMAT_VERSION})")
+                       "(version: ${CLANG_FORMAT_VERSION})")
         message(STATUS "Found cmake-format: ${CMAKE_FORMAT} "
-                "(version: ${CMAKE_FORMAT_VERSION})")
+                       "(version: ${CMAKE_FORMAT_VERSION})")
 
-        # Check if clang-format (in correct version) is available for code formatting.
+        # Check if clang-format (in correct version) is available for code
+        # formatting.
         set(CLANG_FORMAT_REQUIRED "15.0")
         if(NOT (CLANG_FORMAT_VERSION VERSION_EQUAL CLANG_FORMAT_REQUIRED))
             message(FATAL_ERROR "Required clang-format version is "
-                    "${CLANG_FORMAT_REQUIRED}")
+                                "${CLANG_FORMAT_REQUIRED}")
         endif()
     else()
-        message(FATAL_ERROR "UMF_FORMAT_CODE_STYLE=ON, but clang-format or "
+        message(
+            FATAL_ERROR
+                "UMF_FORMAT_CODE_STYLE=ON, but clang-format or "
                 "cmake-format not found (required version: "
                 "${CLANG_FORMAT_REQUIRED})")
     endif()
 
     # Obtain files for clang-format check
     set(format_glob)
     foreach(DIR IN ITEMS include src test benchmark)
-        list(APPEND format_glob
+        list(
+            APPEND
+            format_glob
             "${DIR}/*.h"
             "${DIR}/*.hpp"
             "${DIR}/*.c"
             "${DIR}/*.cpp"
             "${DIR}/**/*.h"
             "${DIR}/**/*.hpp"
             "${DIR}/**/*.c"
-            "${DIR}/**/*.cpp"
-        )
+            "${DIR}/**/*.cpp")
     endforeach()
     file(GLOB_RECURSE format_list ${format_glob})
 
     message(STATUS "Adding clang-format-check and clang-format-apply make "
-            "targets")
-
-    add_custom_target(clang-format-check
-        COMMAND ${CLANG_FORMAT}
-            --style=file
-            --dry-run
-            -Werror
-            ${format_list}
+                   "targets")
+
+    add_custom_target(
+        clang-format-check
+        COMMAND ${CLANG_FORMAT} --style=file --dry-run -Werror ${format_list}
         COMMENT "Check files formatting using clang-format")
 
-    add_custom_target(clang-format-apply
-        COMMAND ${CLANG_FORMAT}
-            --style=file
-            --i
-            ${format_list}
+    add_custom_target(
+        clang-format-apply
+        COMMAND ${CLANG_FORMAT} --style=file --i ${format_list}
         COMMENT "Format files using clang-format")
 
     # Obtain files for cmake-format check
     set(format_cmake_glob)
     foreach(DIR IN ITEMS include src test benchmark)
-        list(APPEND format_cmake_glob 
-            "${DIR}/CMakeLists.txt"
-            "${DIR}/*.cmake"
-            "${DIR}/**/CMakeLists.txt"
-            "${DIR}/**/*.cmake")
+        list(APPEND format_cmake_glob "${DIR}/CMakeLists.txt" "${DIR}/*.cmake"
```

Comment:
hmm... why this list is formatted differently then the one above...? :(
maybe it's different indentation level, but perhaps something can be done...? If not, please ignore this issue